### PR TITLE
Holly iss 1617 properties format yaml

### PR DESCRIPTION
**Pull request by Wyvinar**

### CODE REVIEW

File: go.mod [hunk 1]
```diff
@@ -6,5 +6,6 @@ require (
 	github.com/satori/go.uuid v1.2.0
 	github.com/spf13/cobra v1.1.3
 	github.com/stretchr/testify v1.8.1
+	gopkg.in/yaml.v2 v2.4.0
```
Comment (techcobweb) [inline go.mod:9]: Can we remove this one ? We shouldn't need both ?

File: pkg/utils/ConvertToYaml.go [hunk 2]
```diff
@@ -0,0 +1,18 @@
+/*
```
Comment (techcobweb) [inline pkg/utils/ConvertToYaml.go:1]: We don't need this now right ?

**Review by techcobweb — APPROVED** [review]
Approve on the basis that the comments are addressed.

**Comment by galasa-team** [comment]
Build successful

**Comment by galasa-team** [comment]
Build successful

**Pull request merged**